gcloud-java-examples/README.md

```diff
@@ -89,6 +89,28 @@ To run examples from your command line:
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="download <bucket_name> test.txt"
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="delete <bucket_name> test.txt"
     ```
+ * Here's an example run of `Stat`, illustrating the use of the gcloud-java-nio jar.
+
+     Before running the example, go to the [Google Developers Console][developers-console] to ensure that Google Cloud Storage API is enabled and that you have a bucket with a file in it.
```

Comment:
Actually looks like both entries in the developer's console may need to be enabled ("Google Cloud Storage" and "Google Cloud Storage JSON API).  You can leave what you have as is and I'll update the docs in the master branch after I've done some testing.